kong/dao/schemas/targets.lua

```diff
@@ -0,0 +1,68 @@
+-- This schema defines a sequential list of updates to the upstream/loadbalancer algorithm
+-- hence entries cannot be deleted or modified. Only new ones appended that will overrule
+-- previous entries.
+
+local Errors = require "kong.dao.errors"
+local utils = require "kong.tools.utils"
+
+local default_port = 8000
+local default_weight = 100
+local weight_min, weight_max = 0, 1000
```

Comment:
style: Better spelling constant in UPPERCASE? I thought this was already part of our unformalized codestyle?